dependency-check-maven-config.xml

```diff
@@ -192,4 +217,269 @@
+            apache-karaf-4.1.2.tar.gz: apache-karaf-4.1.2.tar: org.apache.servicemix.bundles.not-yet-commons-ssl-0.3.11_1.jar
+        </notes>
+        <cve>CVE-2004-0009</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to "Apache QPID 0.30" and previous versions. This particular maven artifact is a new qpid artifact.
+            Its first release came after all of these CVEs.
+            qpid-jms-client-0.11.1.jar
+        </notes>
+        <cve>CVE-2015-0223</cve>
+        <cve>CVE-2015-0224</cve>
+        <cve>CVE-2013-1909</cve>
+        <cve>CVE-2012-4460</cve>
+        <cve>CVE-2012-4459</cve>
+        <cve>CVE-2012-4458</cve>
+        <cve>CVE-2012-4446</cve>
+        <cve>CVE-2012-3467</cve>
+        <cve>CVE-2012-2145</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            Applies to an LDAP class that we do not use in the system. Also, this jar is only used in tests.
+            groovy-all-2.4.7.jar
+        </notes>
+        <cve>CVE-2016-6497</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            The CVE applies to xzgrep, not the xz artifact.
+            tika-bundle-1.14.0_2.jar: xz-1.5.jar
+            xz-1.6.jar
+        </notes>
+        <cve>CVE-2015-4035</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs do not apply to this version of bzip2
+            bzip2-0.9.1.jar
+        </notes>
+        <cve>CVE-2011-4089</cve>
+        <cve>CVE-2010-0405</cve>
+        <cve>CVE-2005-1260</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to version 1.7.3 and earlier
+            pax-logging-api-1.10.1.jar
+        </notes>
+        <cve>CVE-2015-1194</cve>
+        <cve>CVE-2015-1193</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            The vulnerability applies to NPM's CLI which does not apply here
+            q-1.4.1.jar
+        </notes>
+        <cve>CVE-2016-3956</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to the Eclipse IDE. OWASP flagged it because eclipse is in the group id.
+            pax-url-aether-2.5.2.jar/META-INF/maven/org.eclipse.aether/aether-spi/pom.xml
+        </notes>
+        <cve>CVE-2010-4647</cve>
+        <cve>CVE-2008-7271</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to the system-level project called spice. OWASP incorrectly labeled this.
+            pax-url-aether-2.5.2.jar/META-INF/maven/org.sonatype.plexus/plexus-cipher/pom.xml:
+            pax-url-aether-2.5.2.jar/META-INF/maven/org.sonatype.plexus/plexus-sec-dispatcher/pom.xml
+        </notes>
+        <cve>CVE-2016-2150</cve>
+        <cve>CVE-2016-0749</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE applies to Apple's Java. The below jar (used by findbugs) is not the same as Apple's Java.
```

Comment:
❓  is this note accurate? I thought we had removed findbugs.

Reply:
This was leftover from the work that started on the 2.11.x branch (where findbugs was **not** removed). Good catch. Removing this block.